blocks/embed/embed.js

```diff
@@ -5,10 +5,95 @@
  */
 import {
   div,
+  span,
 } from '../../scripts/dom-helpers.js';
 import { addFormConfiguration } from '../../scripts/forms-helper.js';
 import { createOptimizedPicture } from '../../scripts/aem.js';
 
+function createLoadingSkeleton() {
```

Comment:
Is there any particular reason we chose this skeletal design for the form? It seems more complex and larger than the actual forms on:

	•	https://issue-283--hubblehomes-com--aemsites.hlx.page/contact-us
	•	https://issue-283--hubblehomes-com--aemsites.hlx.page/warranty

Reply:
its a very similar size to contact-us. The warranty may be smaller. For the skeleton, it doesn't have to match, just be generic

Reply:
Our forms currently place each field on a separate row, while the placeholder puts two inputs on a single row for the first  To maintain consistency, we can adjust the skeletal form to match the basic structure of having one input field per row, just for cosmetic reasons. 

<img width="998" alt="Screenshot 2024-08-08 at 2 14 34 PM" src="https://github.com/user-attachments/assets/78a2f40f-214b-4f94-bac1-4f6ff67093ed">
<img width="878" alt="Screenshot 2024-08-08 at 2 14 49 PM" src="https://github.com/user-attachments/assets/e6d5569a-a620-42c7-bfab-d93652493074">